### PR TITLE
Fix Travis: assume flake8 in Makefile, install it on Travis

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@ before_install:
   - cat /etc/lsb-release
 
 install:
+  - pip install -U flake8
```
Comment (zchee) [inline .travis.yml]: Could you change to the `pip3 install -q -U -r ./tests/requirements.txt`?
It error causes only `--user`, and I want to CI test with hacking modules (more strictly)

Comment (blueyed) [inline .travis.yml]: Done.

**Comment by zchee** [comment]
@blueyed Thanks :)
merge.

**Comment by blueyed** [comment]
@zchee 
Well, it was failing because it also checks jedi.
I've pushed a fixup, but you were too fast.

I'd like to ask you not merge things so eagerly, especially while the CI is failing!

**Comment by zchee** [comment]
@blueyed Ah, sorry :(
Year, you're right \o/

**Pull request merged**